api/v1/flow/flow.pb.go

```diff
@@ -1271,92 +1271,92 @@ type Flow struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	Time *timestamppb.Timestamp `protobuf:"bytes,1,opt,name=time,proto3" json:"time,omitempty"`
+	Time *timestamppb.Timestamp `protobuf:"bytes,1,opt,name=time,proto3" json:"time,omitempty" yaml:"time"`
```

Comment:
Can these yaml tag have `omitempty` as well?

Reply:
I changed yaml parser to `sigs.k8s.io/yaml` which uses json tags for json-vs-yaml consistency.